Assets/PlayerMovement.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] public float moveSpeed = 5f;
```

Comment:
As a general rule, be careful about defaulting values here instead of in Start() as depending on certain actions, this can override the values in editor.